questions/191/explanation.md

```diff
@@ -2,5 +2,6 @@ Due to the `extern "C"` specifications, `A::x` and `B::x` actually refer to the
 
 `x` is first initialized to `0`, then `main()` starts, `0` is printed, `x` is incremented to `1`, and finally `1` is printed.
 
-§[dcl.link]¶6 in the C++ standard:
-"Two declarations for a variable with C language linkage with the same name (ignoring the namespace names that qualify it) that appear in different namespace scopes refer to the same variable."
+§[dcl.link]¶7 in the C++ standard:
+
+> "Two declarations declare the same entity if they (re)introduce the same name, one declares a function or variable with C language linkage, and the other declares such an entity or declares a variable that belongs to the global scope."
```

Comment:
If we use markdown blockquotes (`>`), quotation marks (`"`) are no longer needed.

Reply:
Fixed in 95fac5a94ab16792ce2d9c42b1e0e220eac24525